modules/nw-disabling-multicast.adoc

```diff
@@ -33,6 +33,23 @@ $ oc annotate {namespace} <namespace> \ <1>
 ----
 +
 <1> The `namespace` for the project you want to disable multicast for.
+ifeval::["{context}" == "ovn-kubernetes-disabling-multicast"]
```

Comment:
More just a question that it doesn't need to be included for the other assembly? (Same with the below file that does the same)

Reply:
Great question and I had to review to refresh my memory. The reason it is conditional and for OVN-K is that SDN uses a `Netnamespace` instead of a `Namespace` resource and the `Netnamespace` was not receptive to declarative management.